src/CompanyService/Controllers/DepartmentController.cs

```diff
@@ -2,21 +2,39 @@
 using LT.DigitalOffice.CompanyService.Models.Dto.Models;
 using LT.DigitalOffice.CompanyService.Models.Dto.Requests;
 using LT.DigitalOffice.CompanyService.Models.Dto.Responses;
+using LT.DigitalOffice.Kernel.Responses;
+using Microsoft.AspNetCore.Http;
 using Microsoft.AspNetCore.Mvc;
 using System;
+using System.Net;
 
 namespace LT.DigitalOffice.CompanyService.Controllers
 {
     [Route("[controller]")]
     [ApiController]
     public class DepartmentController : ControllerBase
     {
+        private readonly IHttpContextAccessor _httpContextAccessor;
+
+        public DepartmentController(
+            IHttpContextAccessor httpContextAccessor)
+        {
+            _httpContextAccessor = httpContextAccessor;
+        }
+
         [HttpPost("create")]
-        public Guid Create(
+        public OperationResultResponse<Guid> Create(
             [FromServices] ICreateDepartmentCommand command,
             [FromBody] CreateDepartmentRequest department)
         {
-            return command.Execute(department);
+            var result = command.Execute(department);
+
+            if (result.Status != Kernel.Enums.OperationResultStatusType.Failed)
```

Comment:
Why not use namespace for Kernel.Enums.OperationResultStatusType?

Reply:
done